lang/python/python-libselinux/Makefile

```diff
@@ -47,50 +49,30 @@ define Package/python-libselinux/description
 	This package provides the Python bindings for libselinux.
 endef
 
-# Needed to link libselinux utilities, which link against
-# libselinux.so, which indirectly depends on libpcre.so, installed in
-# $(STAGING_DIR_HOSTPKG).
-HOST_LDFLAGS += -Wl,-rpath="$(STAGING_DIR_HOSTPKG)/lib"
-
-HOST_MAKE_FLAGS += \
-	PREFIX=$(STAGING_DIR_HOSTPKG) \
-	SHLIBDIR=$(STAGING_DIR_HOSTPKG)/lib
-
-$(eval $(call HostBuild))
-
 MAKE_FLAGS += \
 	FTS_LDLIBS=-lfts \
 	SHLIBDIR=/usr/lib \
 	PYTHON=$(PYTHON3) \
 	PYINC="-I $(PYTHON3_INC_DIR)" \
 	PYLIBS="-lpython$(PYTHON3_VERSION)" \
+	$(PYTHON3_VARS)
```

Comment:
Adding `$(PYTHON3_VARS)` fixes the problem of pywrap trying to build using python/host headers.

Reply:
Applied in last push.